Project64.sln

```diff
@@ -1,6 +1,6 @@
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25420.1
+# Visual Studio Version 16

```

Comment:
Note, this change is introduced by the IDE and is inconsequential.
The solution will work the same on any supported VS version.